Frontend web development html css js : Colors in CSS

Rosy03 opened the pull request:
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
<hr>
( I have added the documentation of the topic  selected for contribution "Colors in CSS" via Google Collab. Please review it and merge it. ) <br>
Fixes  #770 

<hr>

## Type of change 📝

<!----Please delete the hashtag from the correct option----->

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

<!----Please delete the hashtag from your domain----->

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [x] Frontend Dev HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

<!----Please delete options that are not relevant. And in order to tick the check box just but x inside them for example [x] like this----->

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

<!----Please delete options that are not relevant. And in order to tick the check box just but x inside them for example [x] like this----->

## Screenshots / Gif (Optional) 📸

<hr>

nidhi333-9 commented:
I have make a document regarding HTML 
[Basics of Internet.md](https://github.com/girlscript/winter-of-contributing/files/7272759/Basics.of.Internet.md)

praveenscience commented:
@nidhi333-9 Why did you add that here?

praveenscience commented:
@Rosy03 Nice attempt, is it possible to convert it into a Markdown file to be added? Just asking.

kashishahuja2002 commented on Web_Development/FrontEnd/Colors-in-CSS_by_RB.md:
```diff
@@ -0,0 +1,9 @@
+# Domain of contribution: FrontEnd with HTML , CSS and JS
+# The Topic : Colors in CSS
+## Issue number that this fixes: #770
```
Remove this line.

kashishahuja2002 commented on Web_Development/FrontEnd/Colors-in-CSS_by_RB.md:
```diff
@@ -0,0 +1,9 @@
+# The Topic : Colors in CSS
+## Issue number that this fixes: #770
+## The Link of the Google Colab for the Documentation :
+   [Colors in CSS by RB.ipnb](https://colab.research.google.com/drive/1HgkiPo41jomjcgWgRXEmVdabrtUWqLz-?usp=sharing)
```
Add the content in this file itself, don't give a link to another file unless it is an audio or a video.

Rosy03 replied:
Sorry but I think in the GWOC Contributions guidelines it was mentioned that the contributors have to submit work by only in Google Collab . Check it please.

kashishahuja2002 commented on Web_Development/FrontEnd/Colors-in-CSS_by_RB.md:
```diff
@@ -0,0 +1,9 @@
+   [Colors in CSS by RB.ipnb](https://colab.research.google.com/drive/1HgkiPo41jomjcgWgRXEmVdabrtUWqLz-?usp=sharing)
+  
+  
+  ## Author name: Rosy Banerjee
```
Remove this line.

kashishahuja2002 commented:
@Rosy03 Also add a reference website for each image.

Rosy03 commented:
> @Rosy03 Also add a reference website for each image.

Excuse me, @kashishahuja2002 I have given picture courtesy for image where necessary.  Other than that, are my own code output which I have added there with the my code itself.

praveenscience commented:
@Rosy03 The content seems to be simple enough, so no need for Google Collab, can you please convert this into Markdown and add the files here? I mentioned this earlier as well. After a brief discussion with the GWoC team and the mentors, I guess we should go with Markdown for this particular PR. Thanks. 😊 

/cc @kashishahuja2002 @2024-SANDHYA

kashishahuja2002 commented:
> > @Rosy03 Also add a reference website for each image.
> 
> Excuse me, @kashishahuja2002 I have given picture courtesy for image where necessary. Other than that, are my own code output which I have added there with the my code itself.

You don't just need to write the name of website. Give proper link of the page where from where you referenced the image.

praveenscience commented:
> > > @Rosy03 Also add a reference website for each image.
> > 
> > 
> > Excuse me, @kashishahuja2002 I have given picture courtesy for image where necessary. Other than that, are my own code output which I have added there with the my code itself.
> 
> Read my review properly. You don't just need to write the name of the website. Give the proper link of the page where from where you referenced the image.

It's alright @kashishahuja2002 Let them do it in MD, maybe they'll do it in a better way. Not sure if Google Collab has those features.

Rosy03 commented:
Hello, @praveenscience sir.
I have did all the possible changes as instructed to me.
Please check it and accept accordingly. I don't think there should be ant further problems now.

kashishahuja2002 commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+
+## Thank you. 
+  
+ ![](images/gwoc_logo_forweb.png " Picture") 
```
Remove this.

kashishahuja2002 commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+  
+ ![](images/gwoc_logo_forweb.png " Picture") 
+  
+ ![](images/img_1_1633004411409.jpg " Picture") 
```
Remove this.

kashishahuja2002 reviewed (changes requested):
There should only be one file (.md), remove all the other binary files. Directly add images in the md file, no need to make separate files for that.

kashishahuja2002 commented:
And also, for the changes suggested by me, you can directly tag me. :)

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+#  **Playing with Colors in CSS**
```
No need of bold in heading.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+## 1. How to define colors in CSS?
+First, we have to know about this. In CSS, when we define properties for different HTML elements, we can give color values as their property in many cases too, like following:
+
+**p {** 
+
+ **color: red;**
+
+**}
```
Use code fences with syntax highlighting.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+ **color: red;**
+
+**}
+//for describing color for paragraph**
```
Use paragraph. `//` doesn't mean anything in documentation.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+
+There are four different ways to specify colors in CSS.
+
+•	“Color keywords” (eg. “Teal”, “Violet”, “Red”, “Yellow” etc.) ![](images/Screenshot%202021-09-28%20072549.jpg " Picture") 
+
+•	Using “RGB” functional notation (“RGB” stands for “Red, Green, Blue model” …. we can define any color in this model), eg. “color: rgb(255, 99, 71);” for “Tomato” color code.
+
+•	Using “HCL” functional notation (“HCL” stands for “Hue, Saturation, Luminosity model” …. we can define any color in this model), eg. “color: rgb(9, 100%,64%);” for “Tomato” color code.
```
Use the proper unordered list in Markdown.
No need of unnecessary empty lines between list items.

praveenscience replied:
Keywords should be in inline code.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+•	Using “RGB” functional notation (“RGB” stands for “Red, Green, Blue model” …. we can define any color in this model), eg. “color: rgb(255, 99, 71);” for “Tomato” color code.
+
+•	Using “HCL” functional notation (“HCL” stands for “Hue, Saturation, Luminosity model” …. we can define any color in this model), eg. “color: rgb(9, 100%,64%);” for “Tomato” color code.
+
```
No need of unnecessary empty lines.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+•	Using “HCL” functional notation (“HCL” stands for “Hue, Saturation, Luminosity model” …. we can define any color in this model), eg. “color: rgb(9, 100%,64%);” for “Tomato” color code.
+
+
+![](images/Screenshot%202021-09-28%20054944.jpg " Picture") 
```
While adding images, please rename correctly.
Add alt text as well. Title can be optional.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+
+![](images/Screenshot%202021-09-28%20054944.jpg " Picture") 
+
+(PC. from w3school.com) [Reference link](https://www.w3schools.com/css/css_colors.asp)
```
Never reference W3Schools. They're of cheap quality.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+Color keywords include the standard primary and secondary colors (such as red, blue, or orange), shades of gray (from black to white, including colors like darkgray and lightgrey), and a variety of other blended colors including lightseagreen, cornflowerblue, and rebeccapurple.
+
+RGB (Red/Green/Blue) functional notation, like hexadecimal string notation, represents colors using their red, green, and blue components. 
+However, instead of using a string, the color is defined using the CSS function **rgb()**. This function accepts as its input parameters the values of the red, green, andblue components.
```
Please add space after and before paragraphs.
Keywords should be in inline code.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+RGB (Red/Green/Blue) functional notation, like hexadecimal string notation, represents colors using their red, green, and blue components. 
+However, instead of using a string, the color is defined using the CSS function **rgb()**. This function accepts as its input parameters the values of the red, green, andblue components.
+
+**red, green, and blue:** Each must be an <integer> value between 0 and 255 (inclusive), or a <percentage>from 0% to 100%. 
```
Keywords should be in inline code.
Add spaces after words, basic grammar.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+**red, green, and blue:** Each must be an <integer> value between 0 and 255 (inclusive), or a <percentage>from 0% to 100%. 
+  
+Hexadecimal string notation represents a color using hexadecimal digits to represent each ofthe color components (red, green, and blue).
+Each color component can be represented as a number between 0 and255 (0x00 and 0xFF) or, optionally, as a number between 0 and 15 (0x0 and 0xF).
```
Add a line space before and after paragraphs.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+  
+Hexadecimal string notation represents a color using hexadecimal digits to represent each ofthe color components (red, green, and blue).
+Each color component can be represented as a number between 0 and255 (0x00 and 0xFF) or, optionally, as a number between 0 and 15 (0x0 and 0xF).
+All components must be specified using the same number of digits.A color in hexadecimal string notation always begins with the character "#". 
```
Add a line space before and after paragraphs.
Use inline code for keywords.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+
+**hsl(hue, saturation, lightness)**
+
+Hue is a degree on the color wheel from 0 to 360. 0 is red, 120 is green, and 240 is blue.
+Saturation is a percentage value, 0% means a shade of gray, and 100% is the full color.
+Lightness is also a percentage, 0% is black, 50% is neither light or dark, 100% is white.
+Designers and artists often prefer to work using the HSL (Hue/Saturation/Luminosity) color method. 
```
Add a line space before and after paragraphs.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+  
+## 2. The ways of making colors translucent in CSS
+  
+To make the desired color into a lighter form we can use “Opacity” as a property for the elements.
+The opacity property specifies the opacity/transparency of an element. It can take a
+value from 0.0 - 1.0. The lower value, the more transparent:  
```
Add a line space before and after paragraphs.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+Lightness is also a percentage, 0% is black, 50% is neither light or dark, 100% is white.
+Designers and artists often prefer to work using the HSL (Hue/Saturation/Luminosity) color method. 
+
+The hsl( )CSS function is very similar to the rgb() function in usage.  
```
Use inline code for keywords.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
````diff
@@ -0,0 +1,469 @@
+Eg.
+
+```  
+.
+.
+.
````
No need of these filler code here.

Rosy03 commented:
Hello. I think I have changed accordingly. 
As for images , I have added them not other way as you have mentioned others in discord I think. [](https://marinegeo.github.io/2018-08-10-adding-images-markdown/) 
Check it please.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+Now, let’s understand about them in brief.
+
+The first and easiest way to specify a color is using one of the 17 predefined color keywords.
+Color keywords include the standard primary and secondary colors (such as red, blue, or orange), shades of gray (from black to white, including colors like darkgray and lightgrey), and a variety of other blended colors including lightseagreen, cornflowerblue, and rebeccapurple.
```
Please add space after and before paragraphs.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+
+![](images/Screenshot%202021-09-29%20091241.jpg " Picture") 
+  
+  (PC. from w3school.com) [Reference link](https://www.w3schools.com/css/css_background.asp)
```
Never reference W3Schools. They're of cheap quality.
Please don't add spaces before starting text.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+value from 0.0 - 1.0. The lower value, the more transparent:  
+  
+
+![](images/Screenshot%202021-09-29%20091241.jpg " Picture") 
```
While adding images, please rename correctly.
Add alt text as well. Title can be optional.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+To make the desired color into a lighter form we can use “Opacity” as a property for the elements.
+The opacity property specifies the opacity/transparency of an element. It can take a
+value from 0.0 - 1.0. The lower value, the more transparent:  
+  
```
No need of unnecessary empty lines.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+Hexadecimal string notation represents a color using hexadecimal digits to represent each ofthe color components (red, green, and blue).
+Each color component can be represented as a number between 0 and255 (0x00 and 0xFF) or, optionally, as a number between 0 and 15 (0x0 and 0xF).
+All components must be specified using the same number of digits.A color in hexadecimal string notation always begins with the character "#". 
+After that come the hexadecimal digits of the color code. The string is case-insensitive. **"#rrggbb"**
```
Use inline code for keywords.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+  
+  (PC. from w3school.com) [Reference link](https://www.w3schools.com/css/css_background.asp)
+  
+We use RGBA model instead of RGB functional notation model and it is also included a fourth component: the alpha channel (or opacity).
+The alpha channel is a number between 0.0 (fully transparent) and 1.0 (fully opaque). We can also specify a percentage where 0% is the same as 0.0 and 100% is the same as 1.0.
+For example, a bright red that's 50% opaque can be represented as rgb(255, 0, 0, 0.5)or rgb(100%, 0, 0, 50%).
```
Add a line space before and after paragraphs.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+The alpha channel is a number between 0.0 (fully transparent) and 1.0 (fully opaque). We can also specify a percentage where 0% is the same as 0.0 and 100% is the same as 1.0.
+For example, a bright red that's 50% opaque can be represented as rgb(255, 0, 0, 0.5)or rgb(100%, 0, 0, 50%).
+
+**In HEX string notation model, we use it as "#rrggbbaa" OR "#rgba" for opacity property.** 
+Specifies a color whose red component is the hexadecimal number 0xrr, green component is 0xgg, and blue component is 0xbb. The alpha channel is specified by0xaa; the lower this value is, the more translucent the color becomes.
```
Add a line space before and after paragraphs.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+**In HEX string notation model, we use it as "#rrggbbaa" OR "#rgba" for opacity property.** 
+Specifies a color whose red component is the hexadecimal number 0xrr, green component is 0xgg, and blue component is 0xbb. The alpha channel is specified by0xaa; the lower this value is, the more translucent the color becomes.
+
+As an example, you can represent the opaque color bright blue as "#0000ff" or "#00f". To make it 25% opaque, one can use "#0000ff44" or "#00f4". 
```
Use inline code for keywords.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+
+As an example, you can represent the opaque color bright blue as "#0000ff" or "#00f". To make it 25% opaque, one can use "#0000ff44" or "#00f4". 
+  
+We can also define “opacity” property for an element in CSS like as following:
```
Use inline code for keywords.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+  
+ }
+ 
+ .
+ .
+ . 
```
No need of these filler code here.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
````diff
@@ -0,0 +1,469 @@
+
+Eg.
+
+```  
````
Use code fences with syntax highlighting.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+  
+We can also define “opacity” property for an element in CSS like as following:
+
+Eg.
```
Use Example.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
````diff
@@ -0,0 +1,469 @@
+ . 
+```
+ Output:
+  
````
No need of unnecessary empty lines.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+ 
+![](images/Screenshot%202021-09-29%200912411.jpg " Picture")
+ 
+  
+  
```
No need of unnecessary empty lines.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+ 
+  
+  
+## 3. The ways of adding colors in CSS
```
Add a line space before and after heading.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+  
+  
+## 3. The ways of adding colors in CSS
+•	**External CSS:** Specifying colors in stylesheets. External styles are defined within the `<link>` element, inside the `<head>` section of an HTML file.
+
+•	**Internal CSS:** Adding all the required properties via `<style>` tag within `<head>` section of an HTML file.
+
+•	**Inline CSS:** An inline style may be used to apply a unique style for a single element.  
```
Use the proper unordered list in Markdown.
No need of unnecessary empty lines between list items.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+  
+# 4.Things that can have color in CSS
+
+At a fundamental level, the color property defines the foreground color of an HTML element's content and the ***background-color*** property defines the element's background color. Thesecan be used on just about any element.
```
Use inline code for keywords.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+
+At a fundamental level, the color property defines the foreground color of an HTML element's content and the ***background-color*** property defines the element's background color. Thesecan be used on just about any element.
+
+## **i) Text**
```
Use proper ordered list and heading in Markdown.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+
+Whenever an element is rendered, these properties are used to determine the color of the text, its background, and any decorations on the text.
+
+**Color:** The color to use when drawing the text and any text decorations (such as the addition of under- or overlines, strike-through lines, and so forth.
```
Use inline code for keywords.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+
+**Color:** The color to use when drawing the text and any text decorations (such as the addition of under- or overlines, strike-through lines, and so forth.
+
+Eg. 
```
Use `Example` in full form not `Eg.`.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+**Color:** The color to use when drawing the text and any text decorations (such as the addition of under- or overlines, strike-through lines, and so forth.
+
+Eg. 
+
+
```
No need of unnecessary empty lines.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
````diff
@@ -0,0 +1,469 @@
+
+
+
+```
+<h1 style="color: Tomato;">Hello World</h1>
+```
````
Use code fences with syntax highlighting.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
````diff
@@ -0,0 +1,469 @@
+<h1 style="color: Tomato;">Hello World</h1>
+```
+Output:  
+  
````
No need of unnecessary empty lines.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+Output:  
+  
+
+![](images/Screenshot%202021-09-28%20132016.jpg " Picture")
```
While adding images, please name correctly.
Add alt text as well. Title can be optional.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+
+![](images/Screenshot%202021-09-28%20132016.jpg " Picture")
+  
+**background-color:** The text's background color.
```
Use inline code for keywords.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+![](images/Screenshot%202021-09-28%20132016.jpg " Picture")
+  
+**background-color:** The text's background color.
+
+
```
No need of unnecessary empty lines.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
````diff
@@ -0,0 +1,469 @@
+
+
+
+```
+<h1 style="background-color:DodgerBlue;">Hello World</h1>
+```
````
Use code fences with syntax highlighting.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
````diff
@@ -0,0 +1,469 @@
+```
+Output:
+  
+![](images/Screenshot%202021-09-28%20132141.jpg " Picture")  
````
While adding images, please name correctly.
Add alt text as well. Title can be optional.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+  
+![](images/Screenshot%202021-09-28%20132141.jpg " Picture")  
+  
+  **text-decoration-color:** By default, text decorations (such as underlines, strikethroughs, etc) use the colorproperty as their colors.
```
Keywords should be in inline code.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+  
+  **text-decoration-color:** By default, text decorations (such as underlines, strikethroughs, etc) use the colorproperty as their colors.
+
+Eg. 
```
Use `Example` in full form not `Eg.`.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+  **text-decoration-color:** By default, text decorations (such as underlines, strikethroughs, etc) use the colorproperty as their colors.
+
+Eg. 
+
+
```
No need of unnecessary empty lines.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
````diff
@@ -0,0 +1,469 @@
+
+
+
+```
````
Use code fences with syntax highlighting.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+  
+![](images/Screenshot%202021-09-28%20152624.jpg " Picture")
+  
+## **ii) Boxes**
```
Use proper ordered list and heading in Markdown.
No need of bold in heading.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+  
+![](images/Screenshot%202021-09-30%20132652.jpg " Picture")
+  
+# 6. Finding the right colors for website 
```
Wrong level of headings.

praveenscience commented on Web_Development/FrontEnd/Colors-In-CSS_by_RB/Playing with Colors.md:
```diff
@@ -0,0 +1,469 @@
+  
+ Making the right choices when selecting colors when designing a website can be a tricky process, especially if you aren't well-grounded in art, design, or at least basic color theory. The wrong color choice can render your site unattractive, or even worse, leave the content unreadable due to problems with contrast or conflicting colors. Worse still, if using the wrong colors can result in your c ...
+
+### That’s the end for today’s lesson about Colors in CSS. 
+### Hope you find this lesson interesting while learning and enjoyed overall. 
+
+
+
+
+## Thank you. 
+  
```
No need of these filler text or story telling tone.

kashishahuja2002 commented:
> Hello @kashishahuja2002 . I think I have changed accordingly. As for images , I have added them not other way as you have mentioned others in discord I think. https://marinegeo.github.io/2018-08-10-adding-images-markdown/ I have done like this too. I don't see it is otherwise. I also see other contributors whose PR have been merged, did same as I did . Check it please.

It doesn't matter how you do it, as long as it's working fine. 
And for others, I have suggested a different method because not everyone is comfortable with this one. You need not worry about others.
